### ISSUE TITLE
Feature request - The user should see the original message in the reply screen

### ISSUE THREAD

**Issue by acquaalta** [learnweb/moodle-mod_moodleoverflow#161]
Hi,
Plugin version: v4.2-r1

When a user replies to a message in the forum, the screen in which the user writes the reply, doesn't show the original message that the user replies to. Therefore, the context of the reply is not shown. So a user that needs the context for the reply, have to  open the original message in one tab and the reply page in another tab. That's cumbersome, of course.
Could you please offer a reply screen that includes the original message?

Please consider it :-)
Thanks in any case, even if just for reading this and considering my request.

**Comment by pejhab** [comment]
Hello. I would say an inline editor would be the way to go. Please see [#83](url).

**Comment by acquaalta** [comment]
Yeah, the proposed solution of an inline editor in #83 seems in line with what we are looking for ;-)
